### ISSUE TITLE
[Bug]: Reka UI 1.0.0-alpha.4 errors in Nuxt

### ISSUE THREAD

**Issue by Robbe95** [unovue/radix-vue#1434]
### Environment

```bash
Developement/Production OS: Windows 10 19043.1110
Node version: 20.0.4
Package manager: pnpm@8.6.0
Radix Vue version: 1.0.0-alpha.4
Vue version: latest
Nuxt version: ^3.14.159
Nuxt mode: universal
Nuxt target: server
Client OS: MacOs
Browser: Chrome 90.0.4430.212
```

### Link to minimal reproduction

https://stackblitz.com/edit/nuxt-starter-rmhreg?file=app.vue

### Steps to reproduce

1. Init a Nuxt project
2. Install reka-ui
3. Use any component

### Describe the bug

Nuxt gets 500 with error `@5.4.2_/node_modules/@vueuse/core/index.js' does not provide an export named 'useVModel'`

### Expected behavior

The component renders

**Comment by zernonia** [comment]
Fixed in `1.0.0-alpha.5`

**Comment by Robbe95** [comment]
Thank you! And for all the awesome work you do!
